README.md

```diff
@@ -8,25 +8,23 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/skn0tt?label=Stay%20updated&style=social)](https://twitter.com/skn0tt)
 
 <p align="middle">
-The Task Queueing Solution for Serverless.
+  The Task Queueing Solution for Serverless.
 </p>
 
 - [Landing Page](https://quirrel.dev)
 - [Documentation](https://docs.quirrel.dev)
 
-Quirrel makes job queueing simple as cake.
-It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
+Quirrel makes job queueing as easy as pie. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
```

Comment:
I'd actually like to keep "simple as cake" here. It always [reminds me of Portal 2](https://en.wikipedia.org/wiki/The_cake_is_a_lie) ;D Also, this is a catchphrase that's spread over a lot of different sources already, so it should be kept consistent :)

```suggestion
Quirrel makes job queueing simple as cake. It supports delayed jobs, fanout jobs, recurring jobs and CRON jobs.
```

Reply:
But the cake was a lie!